### ISSUE TITLE
add-font: generated metadata returns "designer: "UNKNOWN""

### ISSUE THREAD

**Issue by eliheuer** [googlefonts/gftools#112]
When using `add-font` I usually get the following as the second line in the METADATA and need to manually update it:

```
designer: "UNKNOWN"
```

Maybe we could try getting something from the name table?

**Comment by davelab6** [comment]
Perhaps, although it's likely that the value in the metadata file and font
file should be the same.... Maybe better to keep this and in FB tell you
what the ttf value is if metadata is UNKNOWN

**Comment by eliheuer** [comment]
Yeah, a Fontbakery WARN for UNKNOWN designer in the metadata would fix most of the issue here. It's still an extra step add the designer name if it's already specified in the font.

**Comment by davelab6** [comment]
I will transfer this issue to FB repo
